schemas/v1.0/azure.yaml.json

```diff
@@ -525,6 +525,15 @@
                     "$ref": "#/definitions/workflow"
                 }
             }
+        },
+        "cloud": {
+            "type": "object",
+            "title": "The cloud configuration used for the project.",
+            "description": "Optional. Provides additional configuration for deploying to sovereign clouds such as Azure Government.",
+            "additionalProperties": false,
+            "properties": {
+                "name": { "enum": [ "AzureCloud", "AzureChinaCloud", "AzureUSGovernment" ] }
+            }
         }
```

Comment:
Since we don't have a good release story for our schemas consider holding off on this and putting it only in beta for now and then do a smaller PR on release day where we update the v1.0 schema.

Reply:
https://github.com/Azure/azure-dev/pull/3482